src/dynamic_boundary_conditions/rainfall_sites.py

```diff
@@ -0,0 +1,100 @@
+# -*- coding: utf-8 -*-
```

Comment:
Wood be good to add a one line doc-string at the top of these scripts to summarise what they aim to do!

Reply:
Done :) I have also amended the header format/style.